### PR TITLE
UI issues with Positions/Conditions grids

### LINKED ISSUE #499: UI issues with Positions/Conditions grids
Related to #340, #493

1. Add a top border to the table's footer (it is missing when scroll grid to the right)

2. It would be nice to make a columns' values alignment to be the same (left-aligned). Currently on the:
 - conditions list: outcomes and creation date columns' values are right-aligned

 - positions list:  ERC amounts columns' values are right-aligned

3. It would be nice to place 'More' button under the related value (I do not insist on it, just imho)

4. Issue with the sort arrow placement is described here: #281

### PR DESCRIPTION
**Pull request by gabitoesmiapodo**
Closes #499

1. _Add a top border to the table's footer..._ -> Done.
2. _It would be nice to make a columns' values alignment to be the same (left-aligned)..._ -> Left it as it was. Generally speaking, rules for tables are: text is left aligned; dates and numbers are right aligned; everything else (buttons, badges, etc.) is centered. This is the usual or "standard" alignment for tables and data grids (ie: Google spreadsheets).
3. _It would be nice to place 'More' button under..._ -> I changed it for a button / icon.
4. _Issue with the sort arrow placement..._ -> I will try to fix it when I get to #281.

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr535--conditionaltokensexplorer.review.gnosisdev.com

**Comment by elena-zh** [comment]
Hi @gabitoesmiapodo , thank you for the fixes and explanations!
Btw, I have found the issue which is related to the p3: in a position details 'more' button/icon is displayed in different ways now (search for a positionID  = 0xcb9cc334f7ccb5fd103b48281013e822a4d83d0b1532932c7d8ca208b74c7ec6 )
It  would be nice to make icons/buttons look the same.

more

File: src/pages/PositionDetails/Contents.tsx [hunk 1]
```diff
@@ -81,19 +82,6 @@ const CollateralWrapButton = styled(Button)`
...
-  font-size: 14px;
-  font-weight: 600;
-  margin: 0 0 0 12px;
-  text-decoration: underline;
-
-  &:hover {
-    text-decoration: none;
-  }
-`
-
```
Comment (mariano-aguero) [inline src/pages/PositionDetails/Contents.tsx:96]: The `MoreLink` is used in the in this file and throws an error

Screenshot_20201028_094941

**Comment by gabitoesmiapodo** [comment]
@elena-zh @mariano-aguero 

It seems a merge wrecked things up.

It should look like this now:

Screen Shot 2020-10-28 at 17 02 28

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr535--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr535--conditionaltokensexplorer.review.gnosisdev.com

**Review by mariano-aguero — APPROVED** [review]

**Pull request merged**